apps/testingaccessibility/src/components/search/autocomplete.tsx

```diff
@@ -0,0 +1,158 @@
+import React from 'react'
+import Link from 'next/link'
+import {getPathForLesson, getPathForSection} from 'utils/get-resource-paths'
+import {
+  parseAlgoliaHitHighlight,
+  getAlgoliaResults,
+} from '@algolia/autocomplete-preset-algolia'
+import {highlight, searchClient} from 'utils/algolia'
+import {useAutocomplete} from 'hooks'
+import type {Hit} from '@algolia/client-search'
+import {useRouter} from 'next/router'
+import {SearchIcon} from '@heroicons/react/solid'
+
+const Autocomplete = (props: any) => {
+  const inputRef = React.useRef(null)
+  const router = useRouter()
+  const {autocomplete, state} = useAutocomplete({
+    id: 'testing-accessibility-search',
+    defaultActiveItemId: 0,
+    placeholder: 'Search resources',
+    debug: true,
+    navigator: {
+      navigate({itemUrl}) {
+        router.push(itemUrl)
+      },
+    },
+    getSources() {
+      return [
+        {
```

Comment:
something i think that is generally interesting is considering multiple sources for this widget. i don’t know what that would be specifically, but it’s a way to consider where we can broaden search and what “resources” means

in practice it might make sense to simply have them all in a single sanity instance, but is something to noodle on